example/static/disco.html

```diff
@@ -1,349 +1,386 @@
-            Se non hai ancora una identità SPID richiedila ad uno dei gestori.
-          </p>
-	  <p>
-	  <b>eIDAS</b> è il sistema di interoperabilità delle identità digitale europeo. Tramite eIDAS puoi accedere utilizzando il sistema di identità digitale degli altri paesi europei.
-	  </p>
-	  <p class="mb-lg-5">
-	  <b>CIE</b> è il sistema di accesso che consente di utilizzare la propria carta di identità elettronica come identità digitale.
-	  </p>
+  </div>
 
+  <div id="main" class="container my-4">
+    <h4 class="">Benvenuto in Nome Organizzazione Spid Discovery Service</h4>
+    <p><b>SPID</b> è il sistema di accesso che consente di utilizzare, con un'identità digitale unica, i servizi online della Pubblica Amministrazione e dei privati accreditati. Se sei già in possesso di un'identità digitale, accedi con le credenziali del tuo gestore. Se non hai ancora una identità SPID richiedila ad uno dei gestori.</p>
+    <p><b>eIDAS</b> è il sistema di interoperabilità delle identità digitale europeo. Tramite eIDAS puoi accedere utilizzando il sistema di identità digitale degli altri paesi europei.</p>
+    <p><b>CIE</b> è il sistema di accesso che consente di utilizzare la propria carta di identità elettronica come identità digitale.</p>
+    <p class="mb-lg-5"><b>IDEM</b> è la federazione delle identità delle università e degli entri di ricerca.
           <p>
             Seleziona il Provider di Identità presso il quale desideri autenticarti
           </p>
+    <div class="shadow p-3 pb-0 mb-5 bg-white rounded">
+      <div class="row">
+        <div class="col-lg-6 text-center">
+          <div class='ita ita-dropdown ita-l ita-fixed mb-3'>
+            <button class="ita-button">
+              <img src="img/spid.svg" alt="SPID logo">
+              Entra con SPID
+            </button>
+            <div class='ita-menu' role='menu' data-spid-remote></div>
+          </div>
+        </div>
+        <div class="col-lg-6 text-center">
+          <div class='ita ita-l ita-fixed mb-3'>
+            <a class="ita-button" href="/Saml2/disco?entityID=https://idserver.servizicie.interno.gov.it/idp/profile/SAML2/POST/SSO&return=https://sso.isprambiente.it/Saml2/disco">
+              <img src="img/cie.svg" alt="CIE Logo">
+              Entra con CIE
+            </a>
+          </div> 
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-lg-6 text-center">
+          <div class='ita ita-l ita-fixed mb-3'>
+            <a class="ita-button" href="/Saml2/disco?entityID=https://sp-proxy.eid.gov.it/spproxy/idpit&return=https://sso.isprambiente.it/Saml2/disco">
+              <img src="img/ficep.svg" alt="eIDAS Logo">
+              Entra con eIDAS
+            </a>
+          </div>
+        </div>
+        <div class="col-lg-6 text-center">
+          <div class='ita ita-l ita-fixed mb-3'>
+            <a class="ita-button" href="https://wayf.idem.garr.it/WAYF?entityID=_YourEntityID_&return=_YourReturnURL_">
```

Comment:
do we have a note in the docs about this?
_YourEntityID_

Reply:
I'm afraid I didn't add it, I update the pull request with the new documentation.
If is ok, I add a specific md file for the IDEM authentication with the generic informations and the discovery configuration (IDEM Discovery Service Centralized  page)